packages/client/hmi-client/src/petrinet/petrinet-service.ts

```diff
@@ -311,3 +312,77 @@ export const petriToLatex = async (petri: PetriNet) => {
 	const response = await API.post('/transforms/acset-to-latex', petri);
 	return response.data;
 };
+
+/*
+Given two IGraphs, clone the first X times where X is the length of nodes of the 2nd one.
+	Example, SIR + QNQ leads to two distinct SIR graphs. One with Q and one with NQ
+return a graph
+TODO: https://github.com/DARPA-ASKEM/Terarium/issues/868
+*/
```

Comment:
Let's use [jsdoc](https://jsdoc.app/about-getting-started.html) formatting of function comment to standardize them. We don't need to add @params and @return, etc. but more like
```js
/**
 * Given two IGraphs, clone the first X times where X is the length of nodes of the 2nd one.
 * Example, SIR + QNQ leads to two distinct SIR graphs. One with Q and one with NQ
 * @return graph
 * @todo https://github.com/DARPA-ASKEM/Terarium/issues/868
 */
```